hack/make-rules/test-cmd-util.sh

```diff
@@ -2678,6 +2678,38 @@ run_rc_tests() {
   # Clean up
   kubectl delete deployment nginx-deployment-resources "${kube_flags[@]}"
 
+  ## Set probe of a deployment
```

Comment:
maybe add a `dry-run` or a `--local` test

Reply:
have add this in L2685